SparkPost/developers.sparkpost.com
# Resolve dependabot alerts for developers.sparkpost.com

### What Changed

- Resolve dependabot alerts for developers.sparkpost.com
    - Upgrade jpeg-js to 0.4.3
    - Upgrade extend to 3.0.2
    
### How to test

    - Verify with `npm list jpeg-js` that depandabot alert is resolved
    - Verify with `npm list extend` that depandabot alert is resolved
